src/blueprints/tv2/factories/tv2-atem-video-switcher-timeline-factory.ts

```diff
@@ -7,14 +7,15 @@ import { Tv2AtemLayer } from '../value-objects/tv2-layers'
 import { DeviceType } from '../../../model/enums/device-type'
 
 export class Tv2AtemVideoSwitcherTimelineFactory implements Tv2VideoSwitcherTimelineObjectFactory {
-  public createDownstreamKeyerTimelineObject(downstreamKeyer: Tv2DownstreamKeyer, layer: string, onAir: boolean): AtemDownstreamKeyerTimelineObject {
+  public createDownstreamKeyerTimelineObject(downstreamKeyer: Tv2DownstreamKeyer, onAir: boolean): AtemDownstreamKeyerTimelineObject {
+    const downstreamKeyerNumber: string = String(downstreamKeyer.Number + 1)
     return {
       id: '',
       enable: {
         while: 1
       },
       priority: 10,
-      layer,
+      layer: `${this.getDownstreamKeyerLayerPrefix()}_${downstreamKeyerNumber}`, // Taken from Blueprints.
```

Comment:
Don't need to add `Taken from blueprints` so a many times. All of this is taken from Blueprints ;)

Reply:
Have removed the comment.